[32mINFO    [0m integration.ha.test_backups:test_backups.py:490 Remove s3-credentials relation
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:502 Re-add s3-credentials relation
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [executing] maintenance: Configuring backup service...
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:511 Syncing credentials for microceph
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
  s3-integrator/0 [idle] active: 
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.241.98.215:9200/_recovery?human
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=333, count=334, last_expected_id=333)
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.241.98.215:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.241.98.215:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.241.98.215:9200/series_index/_count
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.241.98.215:9200/_recovery?human
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.241.98.215:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.241.98.215:9200/series_index/_count
[32mINFO    [0m integration.ha.helpers:helpers.py:591 Testing restore for series_index - original count pre-restore: 334, and now, new count: 305
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.241.98.215:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.241.98.215:9200/series_index/_count
[32mINFO    [0m integration.ha.conftest:conftest.py:39 



The writes have been cleared.




[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  01:59:27Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active      2  opensearch                                 1  no       
main                               active      1  opensearch                                 0  no       
opensearch                         active      1  opensearch                                 2  no       
s3-integrator                      active      1  s3-integrator             latest/edge     46  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
failover/0                   active    idle   4        10.241.98.54    9200/tcp  
failover/1*                  active    idle   5        10.241.98.239   9200/tcp  
main/0*                      active    idle   2        10.241.98.215   9200/tcp  
opensearch/0*                active    idle   3        10.241.98.37    9200/tcp  
s3-integrator/0*             active    idle   1        10.241.98.159             
self-signed-certificates/0*  active    idle   0        10.241.98.72              

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.241.98.72   juju-7c623d-0  ubuntu@22.04      Running
1        started  10.241.98.159  juju-7c623d-1  ubuntu@22.04      Running
2        started  10.241.98.215  juju-7c623d-2  ubuntu@22.04      Running
3        started  10.241.98.37   juju-7c623d-3  ubuntu@22.04      Running
4        started  10.241.98.54   juju-7c623d-4  ubuntu@22.04      Running
5        started  10.241.98.239  juju-7c623d-5  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-1: 01:47:11 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 01:47:11 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:47:11 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:47:11 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-s3-integrator-0: 01:47:11 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-self-signed-certificates-0: 01:47:11 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 01:47:30 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 01:47:30 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 01:47:30 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-5: 01:47:30 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 01:47:30 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-1: 01:47:30 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 01:47:31 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 01:47:31 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-main-0: 01:47:31 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 01:47:33 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 01:47:33 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-0: 01:47:33 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-failover-0: 01:48:36 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:48:37 ERROR unit.failover/0.juju-log upgrade-version-a:9: [Errno 111] Connection refused
unit-failover-1: 01:48:38 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:48:38 ERROR unit.failover/0.juju-log upgrade-version-a:9: [Errno 111] Connection refused
unit-failover-1: 01:48:39 ERROR unit.failover/1.juju-log node-lock-fallback:8: [Errno 111] Connection refused
unit-failover-0: 01:48:39 ERROR unit.failover/0.juju-log node-lock-fallback:8: [Errno 111] Connection refused
unit-failover-1: 01:48:39 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:48:40 ERROR unit.failover/0.juju-log upgrade-version-a:9: [Errno 111] Connection refused
unit-failover-1: 01:48:40 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:48:41 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:48:41 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:41 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:41 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:41 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:42 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:48:42 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:48:42 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:42 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:43 ERROR unit.failover/1.juju-log upgrade-version-a:9: [Errno 111] Connection refused
unit-failover-0: 01:48:43 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:48:44 ERROR unit.failover/1.juju-log upgrade-version-a:9: [Errno 111] Connection refused
unit-failover-0: 01:48:44 ERROR unit.failover/0.juju-log node-lock-fallback:8: [Errno 111] Connection refused
unit-failover-1: 01:48:45 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:45 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log node-lock-fallback:8: [Errno 111] Connection refused
unit-failover-0: 01:48:46 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:46 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:46 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:46 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:46 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:46 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:48:46 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:46 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:48:47 ERROR unit.failover/1.juju-log upgrade-version-a:9: [Errno 111] Connection refused
unit-failover-0: 01:49:01 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:01 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:02 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 01:49:02 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:03 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:03 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:03 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:03 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:49:03 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-1: 01:49:03 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:04 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-0: 01:49:04 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:49:04 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:49:04 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:49:04 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-1: 01:49:04 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:49:05 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:49:05 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:49:05 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:49:05 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-opensearch-0: 01:49:09 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:49:09 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:49:10 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-main-0: 01:49:10 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 01:49:10 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 01:49:10 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 01:49:10 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:49:11 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-main-0: 01:49:11 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:12 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-main-0: 01:49:12 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 01:49:12 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 01:49:12 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 01:49:12 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:49:13 ERROR unit.opensearch/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-main-0: 01:49:13 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-opensearch-0: 01:49:13 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-main-0: 01:49:14 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-opensearch-0: 01:49:14 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 01:49:15 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-opensearch-0: 01:49:15 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-1: 01:49:15 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-opensearch-0: 01:49:16 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-main-0: 01:49:17 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:49:17 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-main-0: 01:49:18 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:49:18 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:18 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:18 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:18 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-main-0: 01:49:19 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:49:19 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-main-0: 01:49:20 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:20 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-main-0: 01:49:20 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-main-0: 01:49:20 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: Cannot connect to the OpenSearch server...
unit-main-0: 01:49:20 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-main-0: 01:49:20 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-main-0: 01:49:20 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: Cannot connect to the OpenSearch server...
unit-main-0: 01:49:20 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-main-0: 01:49:20 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:49:21 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:22 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-opensearch-0: 01:49:23 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-main-0: 01:49:23 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-main-0: 01:49:23 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:49:24 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:24 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:24 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:24 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-main-0: 01:49:26 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-main-0: 01:49:26 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: Cannot connect to the OpenSearch server...
unit-main-0: 01:49:29 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-main-0: 01:49:29 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:49:33 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:34 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:35 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:35 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:35 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:35 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:35 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:35 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:35 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:37 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:38 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:38 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:38 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:38 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:39 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:39 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:39 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:39 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:50 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:50 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:51 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:51 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:49:53 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:53 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:53 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:53 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:54 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:55 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:55 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:55 ERROR unit.opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 01:49:56 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-0: 01:49:57 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:57 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:57 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:57 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:57 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:58 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:58 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:49:58 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 01:49:59 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:50:00 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:50:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:50:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:02 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:02 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:02 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:02 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:50:02 ERROR unit.opensearch/0.juju-log peer-cluster:11: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:50:03 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 01:50:05 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:05 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:05 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:05 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:05 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 01:50:06 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 01:50:06 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 01:50:06 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:50:06 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:06 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:07 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:07 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:07 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:08 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:50:08 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:50:08 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:50:08 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 01:50:09 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 01:50:09 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:50:09 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:50:10 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:50:10 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 01:50:10 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:10 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:10 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:50:10 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:50:11 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:11 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:50:12 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 01:50:12 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:50:12 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:50:13 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:50:13 ERROR unit.failover/0.juju-log certificates:14: Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:13 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:50:13 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:50:13 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:50:13 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:50:13 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:14 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:14 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:15 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:15 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:15 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:50:15 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 01:50:15 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:15 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:15 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:15 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:16 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:16 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:16 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:16 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:16 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:16 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:16 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:17 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:17 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:17 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:17 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:17 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:17 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:17 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:17 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:17 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:17 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:17 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:17 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:17 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:18 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:18 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:18 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:18 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:18 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:18 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:50:18 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:18 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:18 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:18 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:19 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:19 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:19 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:19 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:19 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:19 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:19 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:19 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:50:19 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:20 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:20 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:20 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:20 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:20 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:21 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:21 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:21 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:50:21 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:21 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:21 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:21 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:22 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:22 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:22 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:22 ERROR unit.failover/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:22 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:22 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:22 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:22 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:22 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:22 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:22 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:22 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:22 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:22 ERROR unit.failover/1.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-1: 01:50:23 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:23 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:23 ERROR unit.failover/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:23 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:23 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:23 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:23 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:24 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:24 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:24 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:24 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:24 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:24 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:26 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:29 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:32 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:35 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:35 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:36 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:37 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:37 ERROR unit.failover/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:37 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:37 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:37 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:37 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:37 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:37 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:37 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:37 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:38 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:38 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:39 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 01:50:39 ERROR unit.failover/1.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:40 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:40 ERROR unit.failover/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:40 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:40 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:40 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:40 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:41 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:41 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:41 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:42 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:42 ERROR unit.failover/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-0: 01:50:42 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:45 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:48 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 01:50:51 ERROR unit.failover/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:52:40 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:52:41 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:52:42 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:52:44 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:52:45 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:52:47 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 01:52:57 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:52:59 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:53:00 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:53:02 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:53:03 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:53:05 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 01:53:18 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:53:19 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:53:20 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:53:22 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:53:23 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-failover-1: 01:53:29 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:53:29 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:53:30 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:53:31 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:53:32 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:53:32 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:53:33 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:53:33 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:53:34 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:53:34 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:53:36 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:53:36 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-main-0: 01:56:25 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:26 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:27 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:28 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:29 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:40 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:42 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:43 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:44 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository
unit-main-0: 01:56:45 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.241.98.215:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.88', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/f813891d-540d-4c1f-b96d-9e8a795a2a08', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.241.98.215:9200/_snapshot/s3-repository

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...
[32mINFO    [0m integration.ha.test_backups:test_backups.py:173 Cleaning backups from cloud buckets